client/src/app/site/pages/meetings/services/meeting-settings-definition.service/meeting-settings-definitions.ts

```diff
@@ -380,6 +421,46 @@ export const meetingSettings: SettingsGroup[] = fillInSettingsDefaults([
                         type: `integer`
                     }
                 ]
+            },
+            {
+                label: _(`Point of order categories`),
+                settings: [
+                    {
+                        key: `list_of_speakers_enable_point_of_order_categories`,
+                        label: _(`Enable point of order categories`),
+                        type: `boolean`
+                    },
+                    {
+                        key: `point_of_order_category_ids`,
+                        label: _(`Point of order categories`),
+                        type: `ranking`,
+                        useRelation: true,
+                        transformFn: (value?: ViewPointOfOrderCategory[]) =>
+                            (value ?? []).map(element => ({
+                                entry: element.text,
+                                allocation: element.rank,
+                                id: element.id
+                            })),
+                        reverseTransformFn: (value: AllocationBox[], original?: ViewPointOfOrderCategory[]) => {
+                            const originalIds = original.map(value => value.id);
+                            const updatedIds = value.map(val => val.id).filter(id => !!id);
+                            const toDelete = originalIds.difference(updatedIds);
+                            const toUpdate: Partial<PointOfOrderCategory>[] = value
+                                .map(val => ({ text: val.entry, rank: +val.allocation, id: val.id }))
+                                .filter(val => {
+                                    if (!val.id) {
+                                        return false;
+                                    }
+                                    const former = original.find(value => value.id === val.id);
+                                    return former.text !== val.text || former.rank !== val.rank;
+                                });
+                            const toCreate: { text: string; rank: number }[] = value
+                                .filter(val => !val.id)
+                                .map(val => ({ text: val.entry, rank: +val.allocation }));
+                            return { toDelete, toUpdate, toCreate };
+                        }
```

Comment:
I am not sure if this should be part of the field definition. Maybe introduce a new field type instead of the transform function. 
If that is too much work or would result in a even more hacky solution I would be fine with the current solution too.